copy/copy.go

```diff
@@ -0,0 +1,165 @@
+package copy
```

Comment:
I would rather have this in `package image` to be `image.Copy(...)` - this way `copy.Image(...)` isn't that beautiful :P

any downsides in having this under `image`?